bob-api/api/de_onderwijzers_api.php

```diff
@@ -0,0 +1,195 @@
+<?php namespace api;
+
+use ErrorException;
+
+/**
+ * Class DeOnderwijzers
+ *
+ * Wrapper for de API die de onderwijzers beschikbaar hebben gesteld.
+ *
+ * Documentatie die goed is om te lezen:
+ * - https://graphql.org/learn/queries
+ * - https://graphql.org/learn/queries/#variables
+ *
+ * Als je debug logging aan wilt hebben, zet dan $DEBUG_ENABLE naar true
+ *
+ * GraphiQl docs: http://api-test.edwh.nl/graphql/?api_key=IBFiOq8Fyoxcqxo4gyDT7pBXzC0REahqcOXu1Nql
+ *
+ * @package api
+ */
+class DeOnderwijzers
+{
+    public static $DEBUG_ENABLED = false;
+
+    /**
+     * The API key to use during the connection
+     */
+    const API_KEY = "IBFiOq8Fyoxcqxo4gyDT7pBXzC0REahqcOXu1Nql";
+
+    /**
+     * The base url of the external connection
+     */
+    const BASE_URL = "http://api-test.edwh.nl/graphql/?api_key=";
+
+    /**
+     * Session token for the application. This should only be used for non-user specific actions.
+     * Once we have a user, his/her sessionToken should be used for subsequent requests.
+     */
+    private $applicationSessionToken;
+
+    /**
+     * DeOnderwijzers constructor.
+     *
+     * Create a new session token that will be used for all subsequent requests
+     * @throws ErrorException
+     */
+    public function __construct()
+    {
+        $this->applicationSessionToken = $this->generateSessionToken();
+    }
+
+    /**
+     * Get information about the currently logged in user
+     * @param string $sessionToken Session token for a user
+     * @return array
+     * @throws ErrorException
+     */
+    public function getMe(string $sessionToken): array
```

Comment:
Probeer specifieker te zijn bij naamgebruik, ik kan niet uit de naam van de functie de functionaliteit halen

Reply:
Goed om hier de volgende keer een suggestie bij te doen. Zeggen dat iets niet goed is is makkelijk, maar hoe moet het dan wel? Zo voorkom je ook dat ik het nu ga aanpassen en jij de volgende keer het nog niet goed vindt. 

Verder, probeer te starten met wat jij vind en daarna pas de oplossing: 'Ik kan niet uit de naam van de functie de functionaliteit halen, kan je de naam specifieker maken? Misschien getCurrentlyLoggedInUserInformation()?'